pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -622,6 +629,11 @@ public Run mapRow(ResultSet rs, int rowNum) throws SQLException {
       r.setRunDirectory(rs.getString("filePath"));
       r.setRunBasesMask(rs.getString("runBasesMask"));
       r.setSequencingParameters(rs.getString("sequencingParameters"));
+      String rawWorkflowType = rs.getString("workflowType");
+      if (!rs.wasNull()) {
```

Comment:
You can just us `r.setWorkflowType(rs.getString("workflowType"))`. It will propagate null correctly:
https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#getString(java.lang.String)
The `wasNull` thing is only necessary for things that return primitive types.

Reply:
The raw value from the query is the MISO enum constant name and is not what we want to use. It gets mapped on the next couple of lines and the raw value is only used in the case of an unknown/unexpected workflow type.

The use of rs.wasNull here is the same as a null check on `rawWorkflowType`